man/ostree.repo-config.xml

```diff
@@ -130,9 +130,10 @@ Boston, MA 02111-1307, USA.
      <varlistentry>
         <term><varname>min-free-space-size</varname></term>
         <listitem><para>Value (in MB, GB or TB) that specifies a minimum space in the
-        underlying filesystem to keep free. Also, note that min-free-space-percent
-        and min-free-space-size are mutually exclusive. Examples of acceptable values:
-        500MB, 1GB etc. The default value is 0MB, which disables this check.
+        underlying filesystem to keep free. Examples of acceptable values: 500MB, 1GB
+        etc. If this option is unset, min-free-space-percent is used instead.</para>
```

Comment:
I think, this line might confuse the user.

It looks like this bit of related information can be mentioned with `min-free-space-percent`'s default value part:
`The default value is 3. This default value is enforced when none of min-free-space-* options are set.`
 And we can get rid of the line above.